operations/workspace/deployment/pkg/common/cluster.go

```diff
@@ -46,3 +53,18 @@ type Overrides struct {
 	// RetryAttempt is used to specify maximum retry attempts that can be made if error occurs
 	RetryAttempt int
 }
+
+var seededRand *rand.Rand = rand.New(
```

Comment:
nit: should use `crypto/rand` imho because the token is used in a security-relevant setting